src/app/Player.js

```diff
@@ -0,0 +1,11 @@
+export class Player {
+  constructor() {}
+
+  askQuestion(question, onQuestionAsked) {
+    onQuestionAsked(question);
```

Comment:
Hmmm... jeszcze tak pomyślałem - co robimy w przypadku kiedy onQuestionAsked bylby undefined?

Reply:
czemu miałbym być undefined? Kiedy mogłaby się zdarzyć taka sytuacja?

Mogę zamknąć to w bloku try catch, pytanie czy to miałeś na myśli?

Reply:
w sumie bardziej jakiegoś if'a niż blok try catch*
@nowakprojects

Reply:
Tak ifa :) No bo w sumie moze być tak, że akurat nikt nie nasłuchuje (przekazuje callback), a nie chcemy, żeby wtedy leciał błąd. I do tego wtedy test. Że nic się nie wydarzy, i nie pójdzie exception.

Reply:
tylko tyle :-) ? 
[commit](https://github.com/nowakprojects/CodersCamp2020.Project.JavaScript.StarWarsQuiz/pull/37/commits/c50739e4f709164c1e3edd4fd43e7fd76220a4c5)

Reply:
Tak, tylko jest specjalna funkcja na przetestowanie, że nie został rzucony wyjątek :) Bo teraz to w sumie nie wiem co testujesz:

https://stackoverflow.com/questions/49603338/how-to-test-an-exception-was-not-thrown-with-jest/49603347

Reply:
Najlepiej też testy odpal sobie jako 
```
jest --coverage
```

i wtedy widzisz jakie linijki masz nie pokryte.

Reply:
Poprawione... tylko pytanie jeszcze takie - test ma sprawdzać, czy funkcja askQuestion nie rzuca wyjątku, czy callback onQuestionAsked? Ja rozumiem, że to pierwsze, bo ona używa onQuestionAsked, które może być undefined...

coverage mam 100%